frontend/smart-kickers-game/src/components/Game/GameStatistics/GameStatistics.js

```diff
@@ -44,27 +28,10 @@ function GameStatistics({ finalScores, onNewGameRequested }) {
         <em>Statistics</em>
       </h2>
       <div className="table-with-stats">
-        <div className="table-item">
-          <FontAwesomeIcon className="blue-team-icon" icon="fa-person" />
-          Blue
-        </div>
-        <div className="table-item"></div>
-        <div className="table-item">
-          <FontAwesomeIcon className="white-team-icon" icon="fa-person" />
-          White
-        </div>
-        <div className="table-item">{finalScores.blue}</div>
-        <div className="table-item">score</div>
-        <div className="table-item">{finalScores.white}</div>
-        <div className="table-item">{returnFastestShot(TeamID.Team_blue)}</div>
-        <div className="table-item">fastest shot of the game</div>
-        <div className="table-item">{returnFastestShot(TeamID.Team_white)}</div>
-        <div className="table-item">{getManualAddedGoals(TeamID.Team_blue)}</div>
-        <div className="table-item">Manually added goals</div>
-        <div className="table-item">{getManualAddedGoals(TeamID.Team_white)}</div>
-        <div className="table-item">{getManualSubstractedGoals(TeamID.Team_blue)}</div>
-        <div className="table-item">Manually substracted goals</div>
-        <div className="table-item">{getManualSubstractedGoals(TeamID.Team_white)}</div>
+        <TeamIcons />
+        <FinalScores blue={finalScores.blue} white={finalScores.white} />
+        <FastestShot blue={TeamID.Team_blue} white={TeamID.Team_white} statistics={statistics} />
```

Comment:
I'd introduce a distinction between those props - they have the same name - `blue` or `white` - but they mean something totally different in context of `FinalScores` vs `FastestShot`.

Wouldn't it be better to
  1) name them `blueScore` and `whiteScore` in `FinalScores`
  2) pass only the `statistics` to `FastestShot`, since `FastestShot` can just import `TeamID` from constants?

Reply:
Names have been changed. Remove unnecessary props. Instead of `statistics` `statistics.fastestShot` was passed.